programmers/문자열-내-p와-y의-개수-세기.js

```diff
@@ -0,0 +1,33 @@
+function solution(s){
+    var count_p = 0;
+//s내부의 p의 개수를 세줄 변수를 선언하였습니다.    
+    var search_p = 'p';
+// indexof라는 함수를 이용하여 p를 찾기 위하여 search_p라는 문자 p를 담고있는 변수를 선언하였습니다. 
+    var index_p = s.indexOf(search_p);
```

Comment:
```suggestion
    const index_p = s.indexOf('p');
```
위에서 문자 'p'를 따로 변수를 선언해 안에 담아서 indexOf함수에 넣어주셨는데  변수를 선언할 필요 없이 그냥 이렇게 넣어주셔도 됩니다